web/client/plugins/Print.jsx

```diff
@@ -74,29 +98,86 @@ import { isInsideResolutionsLimits } from '../utils/LayersUtils';
  *       }
  *    }
  * }
+ *
+ * @example
+ * // Custom plugin added to the left-panel target
+ * {
+ *   "name": "PrintTextInput",
+ *   "id", "ExtraParam",
+ *   "override": {
+ *       "Print": {
+ *          "target": "left-panel",
+ *          "position": 1.5
+ *       }
+ *   },
+ *   "cfg": {
+ *       "property": "extra",
+ *       "label": "Extra stuff",
+ *       "placeholder": "Inserisci cose..."
+ *   }
+ * }
+ *
+ * @example
+ * // Custom plugin replacing an existing target
+ * {
+ *   "name": "PrintTextInput",
+ *   "id", "ExtraParam",
+ *   "override": {
+ *       "Print": {
+ *          "target": "name",
+ *          "position": 1
+ *       }
+ *   },
+ *   "cfg": {
+ *       "property": "extra",
+ *       "label": "Extra stuff",
+ *       "placeholder": "Inserisci cose..."
+ *   }
+ * }
+  *
+ * @example
+ * // Removal of an existing target
+ * {
+ *   "name": "Null",
+ *   "override": {
+ *       "Print": {
+ *          "target": "name",
+ *          "position": 1
+ *       }
+ *   },
+ *   "cfg": {
+ *   }
+ * }
```

Comment:
Instead of having 3 examples and a lot of configurations and code, I suggest to use one single plugin, with no component, using the multi-target support. 
Please try it. 

```suggestion
// PrintCustomizations.js
const MyCustomPanel = () => <div>Hello, I am a custom component</div>
const MyCustomLayout = connect(mapStateToProps, dispatchToProps)(CustomLayoutSelector);
export default  createPlugin('PrintCustomizations',
  containers: [
  // this entry allows to add a new panel between text input and accordion panel
   {
       Print: [{ // this entry add a panel 
            target: "left-panel",
            position: 1.5,
            Component: MyCustomPanel
}, 
// this entry allows to
{
            target: "layout",
            Component: MyCustomLayout
}, {
           target: "map-preview", // To remove one component, simply create a component that returns null.
           Component: () => null 
}]);

```